index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200;300&family=Roboto:ital,wght@0,300;1,500&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="style.css">
+
+    <title>Document</title>
+</head>
+<body>
+    <header class="navbar-container">    
+            <ul>
+                <li><h2 class="logo-item">John Doe</h2></li>
+                <li><img class="handberga" src="Images/Icons/Union.png" alt="handberga menu"></li>
+            </ul>
+    </header>
+
+    <section class="intro-container">
+        <h1>Hey there. I'm Chendi</h1>
+        <h2>I'am a Software Developer</h2>
+        <p>
+            I can help you build a product , feature or website Look 
+            through some of my work and experience! If you like what
+             you see and have a project you need coded, 
+             don't hestiate to contact me.
+        </p>
+        
+       <img src="Images/Social_media.png" alt="social media icons">
+
```

Comment:
- [ ] Kindly make  sure that list of icons uses a ul tag to group them, and li per icon; each icon image is in a link (a tag), currently what you have is a screenshot please fix this 👍🏿